WordPress/src/main/res/values/strings.xml

```diff
@@ -1465,6 +1465,11 @@
         <item>@string/notif_events</item>
     </string-array>
 
+    <!-- likes -->
+    <string name="get_likes_no_network_error">No network available.</string>
+    <string name="get_likes_unknown_error">An unknown error occurred getting likes.</string>
```

Comment:
Maybe we can add `while` (or `when`?) to the message, like this `An unknown error occurred while getting likes.`, to match the tone of similar error messages.

Reply:
👍 ! Definitely better 😄